Wolox/metrics
# Add pre-commit script and refactor test script

### Summary
**Pre-commit script**
- Check if exist ignore-conflict to covertura file.
- Execute test script.

**Test script**
- Remove info about project make it generic.
- Check if the test has passed or failed
- Break long lines
- Add emojis 🤣
